action/node_modules/@actions/core/README.md

```diff
@@ -0,0 +1,146 @@
+# `@actions/core`
```

Comment:
gitignore the `node_modules/` directory.

Reply:
+1 to gitignore the `node_modules`. This would require setting up Node.js and run `npm install` in the workflow like in https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/.github/workflows/ci-workflow.yml

Reply:
You need to include the node_modules file: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github

Reply:
Ok I see. This is unfortunate :(. There are a lot of files.

Reply:
I figure these actions are fairly new, so this is hopefully one of the bugs they'll iron out as time goes on. We should check back to see if there are improvements in the future (and, also, when they officially support writing these in Go, that'd be cool too :)...)